### PR TITLE
Update fulfilment/order management process to include PAYMENT_PENDING status

### PR DESCRIPTION
**Pull request by martinsrenato**
New flow will stop and wait for some minutes if the order is still on PAYMENT_PENDING status

### CODE REVIEW

File: adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckPendingOrderAction.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import de.hybris.platform.processengine.action.AbstractAction;
+import org.apache.log4j.Logger;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class AdyenCheckPendingOrderAction extends AbstractAction<OrderProcessModel> {
+	private static final Logger LOG = Logger.getLogger(AdyenCheckPendingOrderAction.class);
+
+	public enum Transition {
```
Comment (rkewlani) [inline adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckPendingOrderAction.java:36]: can we use existing enums for this?

File: adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckNewOrderAction.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		if (order == null) {
+			LOG.error("Missing the order, exiting the process");
+			return Transition.NOK.toString();
+		}
+
+		if (OrderStatus.PAYMENT_PENDING.equals(order.getStatus())) {
+			LOG.debug("Process: " + process.getCode() + " Order Payment Pending, will wait...");
+			return Transition.WAIT.toString();
+
+		} else if (OrderStatus.CANCELLED.equals(order.getStatus())
```
Comment (rkewlani) [inline adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckNewOrderAction.java:46]: do we need to take care of cancelled here?

File: adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckPendingOrderAction.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		}
+
+		if (OrderStatus.PAYMENT_PENDING.equals(order.getStatus())) {
+			LOG.debug("Process: " + process.getCode()
+					+ " Order still on Payment Pending, will update status to PROCESSING_ERROR...");
+			order.setStatus(OrderStatus.PROCESSING_ERROR);
+			modelService.save(order);
+			return Transition.NOK.toString();
+
+		} else if (OrderStatus.CANCELLED.equals(order.getStatus())
```
Comment (rkewlani) [inline adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckPendingOrderAction.java:69]: same here for cancelled order

File: adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckPendingOrderAction.java (hunk 1 above)
Comment (martinsrenato) [reply adyenv6core/src/com/adyen/v6/actions/order/AdyenCheckPendingOrderAction.java]: Yes, already fixed.

**Pull request merged**